docs/builders/webpack.md

```diff
@@ -128,6 +128,19 @@ Storybook's default Webpack configuration provides support for most project setu
 
 <!-- prettier-ignore-end -->
 
+However, if you're working with a framework that provides a default aliasing configuration (e.g., Next.js, Nuxt) and you want to configure Storybook to use the same aliases, you may not need to install any additional packages. Instead, you can extend the default configuration of Storybook to use the same aliases provided by the framework. For example, to set up an alias for the `@` import path, you can add the following to your `.storybook/main.js|ts` file:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-main-ts-module-resolution-atsign-import.js.mdx',
+    'common/storybook-main-ts-module-resolution-atsign-import.ts.mdx',
+  ]}****
```

Comment:
@noviceGuru can you remove the extra `****` here?

Reply:
@jonniebigodes It's totally fine, sorry it slipped my hand, lost the notification among the emails.
I corrected it, and had to pull before pushing, so it should merge fast forward. Let me know for any other corrections.